src/api-v1.js

```diff
@@ -740,26 +740,64 @@ api.declare({
   route: '/state/:workerType',
   name: 'state',
   title: 'Get AWS State for a worker type',
-  scopes: [['aws-provisioner:view-worker-type:<workerType>']],
+  output: 'state-response.json#',
```

Comment:
I think the calls are two different things -- if you want to know roughly "how this worker type is doing" (meaning, is it pegged at its max capacity? is it busily spinning up new instances?), you call the status endpoint.  If you want to know exactly which resources are associated with it, you call the ec2-objects (or maybe resources?) endpoint.  I expect that the former is going to be much more useful to everyone -- it might appear in treeherder, for example, as an indicator of current load; or be sampled periodically to create some kind of "burn rate" expense dashboard.

I strongly dislike the "internal format" idea -- an API shouldn't expose internal details, and whatever that returns is, by definition, an external representation.  And that representation should make sense to someone not familiar with the implementation.  In this case, it's pretty sensible: there are a set of instances in either the pending or running state, and a set of requests which have not yet been associated with an instance.  I'd actually prefer to put that in a single list of EC2 resources, some of which just have 'null' for the instanceId, indicating that they are un-fulfilled requests.